src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -70,6 +72,7 @@ internal CurlHandler()
                 throw new HttpRequestException(SR.net_http_client_execution_error);
             }
             SetCurlMultiOptions();
+            PollFunction(_multiHandle);
```

Comment:
Do we need to start polling here?  Shouldn't we only need to poll when we know there's something to be polled?  No work has been initiated yet.